_build/data/transport.core.system_settings.php

```diff
@@ -567,7 +567,7 @@
 $settings['feed_modx_security_enabled']= $xpdo->newObject('modSystemSetting');
 $settings['feed_modx_security_enabled']->fromArray(array (
   'key' => 'feed_modx_security_enabled',
-  'value' => '1',
+  'value' => '0',
```

Comment:
Why would you still leave default value as `0`. Your previous PR was closed because of this